src/ConnectedMode/packages.lock.json

```diff
@@ -88,6 +97,12 @@
         "resolved": "13.0.1",
         "contentHash": "ppPFpBcvxdsfUonNcvITKqLl3bqxWbDCZIzDWHzjpdAHRFfZe0Dw9HmA0+za13IdyrgJwpkDTDA9fHaxOrt20A=="
       },
+      "StrongNamer": {
+        "type": "Direct",
+        "requested": "[0.0.8, )",
+        "resolved": "0.0.8",
+        "contentHash": "7GAMIhQSKabPrXVtEpDQoLRWVZUn1IGzeHvfDSvs5lPhU4KPFMQVpI/lDFikhMXr8FmdFBOBGaHEhQ9azMK3TA=="
+      },
```

Comment:
`Microsoft.Alm.Authentication.dll` is unsigned, which means we couldn't reference it directly from our signed assemblies. So, at build time we sign `Microsoft.Alm.Authentication.dll` with our key.

That's what the `StrongNamer` NuGet package is used for - signing the MS assembly at build time. This isn't new - it was added years ago.